### DISCUSSION TITLE
SMAPI 4.1.7.2

### DISCUSSION THREAD

**Discussion by NRTnarathip** [NRTnarathip/SMAPI-Android-1.6#2] [General]
### The issues we are aware of and are working to resolve
- Application Not Respond or Game Crash
- Android 10, 9, 8, 7... error when Install SMAPI Zip 
you must go to app setting & click allow permission file access
- Other many bug

<hr /><em>This discussion was created from the release <a href='https://github.com/NRTnarathip/SMAPI-Android-1.6/releases/tag/smapi'>SMAPI 4.1.7.2</a>.</em>

**Comment by Vndettaa** [comment]
Thats Nice,
Unfortunately the launcher is crash on my Device,
I Hope its Fixed Soon, Good Work!.

**Reply by NRTnarathip** [reply]
what android are you use?

**Reply by Vndettaa** [reply]
14

**Reply by NRTnarathip** [reply]
is crash on click "Start Game"
or click open launcher

**Reply by Vndettaa** [reply]
Opening a launcher

**Reply by NRTnarathip** [reply]
did you have installed game?

**Reply by NRTnarathip** [reply]
launcher it need game latest version

**Reply by Vndettaa** [reply]
Stardew Valley? Ofc

**Reply by Vndettaa** [reply]
Yeah it's latest, i think its just my Phone Problem, Dont worry, keep up your Work!

**Reply by NRTnarathip** [reply]
https://discord.com/invite/ETtycvcJjr
if you have free time please contact me

**Reply by Vndettaa** [reply]
Whats your id mate? In discord, is it Error[123]?
I cant send you a message or Friend Request, my username is Vndetta.

Thanks!

**Comment by Hazelzoo** [comment]
'Smpi launcher isn't responding'
Android 14 original stardew version 1.6.14.6

**Reply by NRTnarathip** [reply]
i will fix this bug in next week
please wait🥲🙏

**Comment by marissafranklin2022-sudo** [comment]
Which one do I download for SMAPI.Launcher.v1.1.4.apk
